telegram/chat.py

```diff
@@ -433,6 +457,52 @@ def ban_member(
             revoke_messages=revoke_messages,
         )
 
+    def ban_sender_chat(
```

Comment:
what about another shortcut `ban_chat` for `bot.ban_chat_sender_chat(sender_chat_id=self.id,…)` so that one can do `message.sender_chat.ban`? same for unban

maybe you have suggestions for better namings. `banChatSenderChat` is a bit weird though - would have named it `banSenderChat` …

Reply:
> what about another shortcut `ban_chat` for `bot.ban_chat_sender_chat(sender_chat_id=self.id,…)` so that one can do `message.sender_chat.ban`? same for unban

In `chat.py`? What if we add this shortcut in `message` instead? So one can do: `message.ban_chat()` which will call `bot.ban_chat_sender_chat(chat_id=self.chat_id, sender_chat_id=self.sender_chat.id)`

Reply:
`Chat.bat_chat` was the idea, yes. I don't think we have any shortcut that passes along attributes of the current classes attributes - e.g. we also don't have `message.ban_chat_member`.
__if__ we want something like that, then I'd argue that it would be better to have it in `Update` (`update.ban_chat_sender_chat = bot.bat_chat_sender_chat(chat_id=update.effective_chat.id, sender_chat_id=update.effective_message.chat.id`). Would you be okay with postponing this idea? We can discuss it separately and if decide to go for it, we can also apply it for other cases that fit this scheme.

Reply:
on second thought, no I guess this isn't a good idea, there will be too many shortcuts / ambiguity / complexity. I'll just go with your initial suggestion

Reply:
all right :)